### PR TITLE
WIP OCPBUGS#9229: Graceful Shutdown improvements

### PR DESCRIPTION
**Pull request by nalhadef**
Version(s):
Main and 4.6 and above

Issue:
[Bug 2076315](https://bugzilla.redhat.com/show_bug.cgi?id=2076315)

Link to docs preview (Updated Sept. 8, 2023):
https://45621--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/graceful-cluster-shutdown#graceful-shutdown_graceful-shutdown-cluster

Additional information:

QE Reviewer: liqcui@redhat.com

### CODE REVIEW

File: modules/graceful-shutdown.adoc [hunk 1]
```diff
@@ -38,42 +38,40 @@ $ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-s
 ----
 <1> To ensure that the cluster can restart gracefully, plan to restart it on or before the specified date. As the cluster restarts, the process might require you to manually approve the pending certificate signing requests (CSRs) to recover kubelet certificates.
 
-. Shut down all of the nodes in the cluster. You can do this from your cloud provider's web console, or run the following loop:
+. Mark the control plane nodes in the cluster as unschedulable. You can do this from your cloud provider's web console, or run the following loop:
```
Comment (lyman9966) [inline modules/graceful-shutdown.adoc:41]: Indeed, from the following command, it not only mark the control plane nodes as unschedulable, but mark all the nodes (workers and control plane nodes). Please update the description correctly.

Comment (nalhadef) [reply modules/graceful-shutdown.adoc:41]: Thanks. @lyman9966 . What would be the correct description?

Comment (nalhadef) [reply modules/graceful-shutdown.adoc:41]: @lyman9966, if you aren't the right person to provide the description, can you point me to someone who could? Thanks!!

Comment (lyman9966) [reply modules/graceful-shutdown.adoc:41]: @nalhadef I think the description should be: Mark all the nodes in the cluster as unschedulable. You can do this from your cloud provider's web console, or run the following loop:

**Comment by nalhadef** [comment]
Sent for QE review September 8, 2023.

File: modules/graceful-shutdown.adoc (hunk 1 above)
Comment (lyman9966) [reply modules/graceful-shutdown.adoc:49]: Hi, @nalhadef 
It seems keep not update. It should be: Mark all the nodes in the cluster as unschedulable.

File: modules/graceful-shutdown.adoc [hunk 2]
```diff
@@ -33,110 +33,56 @@ If your cluster fails to recover, follow the steps to restore to a previous clus
...
-+
-[source,terminal]
 ----
-$ oc debug node/<node_name>
+2022-08-05T14:37:50Zuser@user:~ $ <1>
 ----
+<1> To ensure that the cluster can restart gracefully, plan to restart it on or before the specified date. As the cluster restarts, the process might require you to manually approve the pending certificate signing requests (CSRs) to recover kubelet certificates.
 
-... Change your root directory to `/host` by running the following command:
+. Mark the control plane nodes in the cluster as unschedulable. You can do this from your cloud provider's web console, or run the following loop:
```
Comment (lyman9966) [inline modules/graceful-shutdown.adoc:49]: Hi, @nalhadef 
It seems keep not update. It should be: Mark all the nodes in the cluster as unschedulable.

**Comment by kalexand-rh** [comment]
Closing in favor of https://github.com/openshift/openshift-docs/pull/65094.

**Pull request closed without merging**